pandas/tests/io/conftest.py

```diff
@@ -56,6 +66,9 @@ def s3_resource(tips_file, jsonl_file, feather_file):
         os.environ.setdefault("AWS_SECRET_ACCESS_KEY", "foobar_secret")
 
         moto = pytest.importorskip("moto")
```

Comment:
```suggestion
        moto = pytest.importorskip("moto", minversion="1.3.14")
```

Reply:
I didn't know you could do that, would have saved me keystrokes in a lot of places

Reply:
done